JavaPro13.09.2022.Lesson3

BlackbrownKK opened the pull request:
Class "HomeWorkApp" created in Project JavaPro13.09.2022 on local PC under new branch , previous class "Monobank" had been deleted.

BlackbrownKK committed:
Class Monobank created.

BlackbrownKK committed:
Class Monobank has deleted.

BlackbrownKK committed:
Class HomeWorkApp created.

BlackbrownKK committed:
All clases created.

shaitanaleksander commented on src/HomeWorkApp.java:
```diff
@@ -0,0 +1,175 @@
+
+    // Task 7.
+    public static void plusMinus() {
+        int a;
```
don't separate the declaration and initialization of a variable, it will be better to declare it at line 82

shaitanaleksander commented on src/HomeWorkApp.java:
```diff
@@ -0,0 +1,175 @@
+
+    // Task 8.
+    public static void plusTrueMinusFalse() {
+        int a;
```
same here

shaitanaleksander commented on src/HomeWorkApp.java:
```diff
@@ -0,0 +1,175 @@
+    // Task 9.
+
+    public static void numberOfTimes() {
+        String string = "Lockey is ";
```
name is not descriptive  String string

shaitanaleksander commented on src/HomeWorkApp.java:
```diff
@@ -0,0 +1,175 @@
+    public static void numberOfTimes() {
+        String string = "Lockey is ";
+        int variable = 777;
+        int number;
```
same

shaitanaleksander commented on src/HomeWorkApp.java:
```diff
@@ -0,0 +1,175 @@
+    }
+
+    public static String newLine(String string, int variable, int number) {
+        int a;
```
you can declare it in the for loop, line 128

shaitanaleksander commented on src/HomeWorkApp.java:
```diff
@@ -0,0 +1,175 @@
+
+    // Task 10.
+    public static void leapYear() {
+        int year;
```
same

BlackbrownKK committed:
Changes in methods:
in method inRange(): line 61,63. Initialization and value assignment variable a and b.
in method plusMinus(): line 79. Initialization and value assignment variable.
in method checkPlusMinus():  line 83. Renamed variable.
in method plusTrueMinusFalse(): line 97. Initialization and value assignment variable.
in method checkplusTrueMinusFalse(): line 101. Renamed variable.
in method numberOfTimes(): line 112, 113 Renamed variable. 116 Initialization andvalue assignment variable.
in method newLine(): line 122 Initialization and value assignment variablein the for loop.
